### ISSUE TITLE
PeftConfig missing lora_alpha field — defaults to 8, dampens high-rank LoRA

### ISSUE THREAD

**Issue by anikita** [huggingface/lerobot#3551]
## Problem

`PeftConfig` in `src/lerobot/configs/default.py` defines `r` (rank) but has **no `lora_alpha` field**. The PEFT library's `LoraConfig` defaults `lora_alpha` to 8 regardless of rank. This means:

| Rank (r) | alpha | Scaling (alpha/r) | Effective LR multiplier |
|----------|-------|--------------------|------------------------|
| 8 | 8 | 1.0 | 1.0 (works) |
| 16 | 8 | 0.5 | 0.5 |
| 32 | 8 | 0.25 | 0.25 |
| 64 | 8 | **0.125** | **0.125** |

At r=64, the LoRA contribution is dampened 8x compared to alpha=r. Gradients flowing through the adapter are scaled by 0.125, so the effective learning rate is 1/8th of what the user specified. After 15K steps at r=64 with lr=1e-4, the adapter weights effectively received ~1.9K steps worth of training at lr=1.25e-5.

## Symptom

Robot barely moves at deployment — the base model (`smolvla_base`, untuned for the target embodiment) dominates output, and the LoRA adapter is too weak to overcome it. Same behavior across multiple checkpoints trained with different data (confirmed on SO-101 arm).

## Verification

1. Trained SmolVLA LoRA r=64 → robot near-static
2. Edited cached `adapter_config.json`: `"lora_alpha": 8` → `"lora_alpha": 64`
3. Re-ran deployment → robot moved immediately (overshooting, because weights were trained under 0.125x scaling but inferred at 1.0x)

## Fix

One-line addition to `PeftConfig`:

```python
# In src/lerobot/configs/default.py, after r: int = 16
lora_alpha: int = 8
```

This keeps backward compatibility (default 8) while allowing users to set `--peft.lora_alpha=64` (or any value) at training time. The field is already a standard `LoraConfig` parameter — it just needs to be exposed through LeRobot's config layer.

## Related

- #3458 (factory.py PosixPath bug)
- #3459 (train.py double PEFT wrapping)
